index.Rmd

```diff
@@ -20,116 +21,102 @@ a:visited {color: #91170a;}
 
 ```{r, echo=FALSE, message=FALSE}
 library(knitr)
-opts_chunk$set(fig.width=5, fig.align='center', fig.height=5, dpi=72)
+opts_chunk$set(fig.width = 12, fig.align = 'center', fig.height = 8, dpi = 72)
 ```
-<!--
-# Today's items
 
+# Week 2, Tutorial 6: Advanced Raster Analysis
 
-Good morning!
-Here is what you will do today:
-
-Time  | Activity
-------------- | -------------
-Until 11:00 |  	Review answer of the other team for yesterday’s exercise
-Morning  |  Self-study: go through the following tutorial
-14:00 to 14:40  | Presentation and discussion
-Rest of the afternoon | Do/finalise the exercise.
--->
-
-# Week 2, Lesson 6: Advanced Raster Analysis
-
-## Learning outcomes of today:
-
-Being able to:
-
-  - carry out a supervised classification on a series of raster layers
-  - construct a raster sieve using the `clump()` function
-  - deal with thematic (categorical maps)
+## Learning objectives
+* Carry out a supervised classification on a SpatRaster
+* Construct a raster sieve using the `patches()` function
+* Deal with thematic (categorical maps)
 
 # Advanced Raster Analysis
 
 ## Introduction to Sentinel-2 data used here
 
-We will carry out a supervised classification using Sentinel 2 data for Gewata region in Ethiopia. Athmospherically corrected Level 2A data acquired on December 27 2020 used in this excercise. The data is download from ESA's online data hub (https://scihub.copernicus.eu/dhus), a part of the Copernicus European Programme. As it is freely available, Sentinel data has been commonly used next to Landsat data for environmental monitoring.
+We will carry out a supervised classification using Sentinel 2 data for the Gewata region in Ethiopia. To do this, we use atmospherically corrected Level 2A data acquired on December 27 2020. This data was downloaded from ESA's online data hub (https://scihub.copernicus.eu/dhus), a part of the Copernicus European Programme. As it is freely available, Sentinel data has been commonly used next to Landsat data for environmental monitoring.
```

Comment:
Apparently in (regular rather than computer) science, "data" is plural, so we're supposed to say "these data were downloaded". A single data point is "datum".